fs2-kafka/src/main/scala/com/avast/sst/fs2kafka/Fs2KafkaModule.scala

```diff
@@ -0,0 +1,92 @@
+package com.avast.sst.fs2kafka
+
+import cats.effect.{Blocker, ConcurrentEffect, ContextShift, Resource, Timer}
+import fs2.kafka._
+
+object Fs2KafkaModule {
+
+  def makeConsumer[F[_]: ConcurrentEffect: ContextShift: Timer, K: Deserializer[F, *], V: Deserializer[F, *]](
```

Comment:
this should have an arg for the consumer provider
If I recall it's of type 
```
type ConsumerProvider[F[_]] = Map[String, String] => F[KafkaByteConsumer]
```

used with `.withCreateConsumer(...)` fs2-kafka prop. This is very useful for tests, since one can pass a `MockConsumer`